sync2jira/downstream_issue.py

```diff
@@ -977,12 +980,27 @@ def _update_github_project_fields(client, existing, issue, github_project_fields
     """
 
     for name, values in github_project_fields.items():
-        _, jirafieldname = values
-        try:
-            existing.update({jirafieldname: str(getattr(issue, name))})
-        except JIRAError as err:
-            # Add a comment to indicate there was an issue
-            client.add_comment(existing, f"Error updating GitHub project field: {err}")
+        fieldvalue = getattr(issue, name)
+        if name == 'storypoints':
+            try:
+                jirafieldname = default_jira_fields['storypoints']
+            except KeyError:
+                log.error("Configuration error: Missing 'storypoints' in `default_jira_fields`")
+                raise Exception
+            try:
+                existing.update({jirafieldname: fieldvalue})
+            except JIRAError as err:
+                # Add a comment to indicate there was an issue
+                client.add_comment(existing, f"Error updating GitHub project storypoints field: {err}")
+        elif name == 'priority':
+            jira_priority = github_project_fields['priority']['options'].get(fieldvalue)
+            if not jira_priority:
+                jira_priority = ''
+            try:
+                existing.update({'priority': {'name': jira_priority}})
+            except JIRAError as err:
+                # Add a comment to indicate there was an issue
+                client.add_comment(existing, f"Error updating GitHub project priority field: {err}")
```

Comment:
Is there unit test coverage for this code?

Reply:
Done. 7db5135c032ea8c8a5684d9b108863acc9a5e5a5